client/src/components/TaskStatusChart/TaskStatusChart.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import styles from "./TaskStatusChart.module.css";
+import { PieChart, Pie, Cell, Legend } from "recharts";
+import { PropTypes } from "prop-types";
+
+const RADIAN = Math.PI / 180;
+const renderCustomizedLabel = ({
```

Comment:
I would place this inside the `TaskStatusChart` function

Reply:
I would also just rename the function to `chartLabel`

Reply:
Resolved: I made this change.